Spinit.CosmosDb/Internals/CosmosDbCollection.cs

```diff
@@ -143,19 +143,19 @@ public async Task<int> CountAsync(ISearchRequest<TEntity> request)
             return result.Resource;
         }
 
-        public Task<int?> GetThroughputAsync()
+        public async Task<ThroughputProperties> GetThroughputAsync()
         {
-            return _container.ReadThroughputAsync();
+            return await _container.ReadThroughputAsync(new RequestOptions()).ConfigureAwait(false);
         }
 
-        public Task SetThroughputAsync(int throughput)
+        public Task SetThroughputAsync(ThroughputProperties throughputProperties)
         {
-            if (!ThroughputValidator.IsValidThroughput(throughput))
+            if (throughputProperties is { } && !ThroughputValidator.IsValidThroughput(throughputProperties, out var message))
```

Comment:
Vad innebär `throughputProperties is { }` ?

Reply:
Vet inte om det strikt är nödvändigt, men vi har väl inte nullable enablat här, så `throughputProperties` kan vara null tänker jag. `is { }` motsvara ju `!= null`, men jag kan ändra till det om du tycker det är mer intuitivt?

Reply:
OK! Nejdå, det blir säkert bra!